live-build/misc/ansible-roles/appliance-build.minimal-common/tasks/main.yml

```diff
@@ -266,3 +266,45 @@
       ## Record command exit failures (execve result is not command result)
       -a exit,always -F a0!=0 -S exit_group
       -a exit,always -F a0!=0 -S exit
+
+#
+# By default, the ulimit for core files is set to 0, and the default
+# filename and location for a core file is 'core' in the cwd. Update
+# limits.conf to allow processes running as root or a regular user to
+# make core files.
+#
+- lineinfile:
+    create: yes
+    dest: /etc/security/limits.conf
+    line: "{{ item.domain }} soft core unlimited"
+  with_items:
+    - { domain: "root" }
+    - { domain: "*" }
+
+#
+# Create a world writeable directory for application core dumps.
```

Comment:
I think it'd be useful to put your analysis in a comment here, about how core dumps are created/written as the user of the process, rather than always using "root" like how illumos does it; which is why we want it to be world writable.